dts/arm/st/f1/stm32f103Xc.dtsi

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2017 I-SENSE group of ICCS
+ *
+ * SoC device tree include for STM32F103xE SoCs
```

Comment:
STM32F103x**C**

Reply:
stm32f103Xc.dtsi is based on previous stm32f103Xe.dtsi. 
A real mistake. Working on it.